src/drivers/displays/tDisplayV1Driver.cpp

```diff
@@ -96,7 +96,6 @@ void tDisplay_MinerScreen(unsigned long mElapsed)
 
   // Push prepared background to screen
   background.pushSprite(0, 0);
-  background.deleteSprite();
```

Comment:
Hi! Not sure if this should be deleted. 
There is a really lack of memory on this devices so must be added